aggregate/perf_session.go

```diff
@@ -27,19 +25,30 @@ func (p *PerfCounter) String() string {
 	return fmt.Sprintf("{Name: %s, Value: %f}", p.Name, p.Value)
 }
 
-// NewPerfSession returns an instance of PerfSession.
-func NewPerfSession(addr string) *PerfSession {
+// NewPerfSessionByAddr returns an instance of PerfSession using node address
+func NewPerfSessionByAddr(addr string) *PerfSession {
 	return &PerfSession{
-		RemoteCmdClient: admin.NewRemoteCmdClient(addr, session.NodeTypeReplica),
-		Address:         addr,
+		NodeSession: session.NewNodeSession(addr, session.NodeTypeReplica),
+	}
+}
+
+// NewPerfSessionByNodeSession returns an instance of PerfSession using existed session.
+func NewPerfSessionByNodeSession(session session.NodeSession) *PerfSession {
```

Comment:
```suggestion
// WrapPerf returns an instance of PerfSession using an existing session.
func WrapPerf(session session.NodeSession) *PerfSession {
```
Go prefers shorter name.